pootle/apps/pootle_score/scores.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django.conf import settings
+
+from pootle_store.constants import FUZZY, TRANSLATED, UNTRANSLATED
+
+SCORE_SUGGESTION_ADD = settings.POOTLE_SCORE_COEFFICIENTS['SUGGESTION_ADD']
+SCORE_SUGGESTION_ACCEPT = settings.POOTLE_SCORE_COEFFICIENTS[
+    'SUGGESTION_ACCEPT']
+SCORE_SUGGESTION_REJECT = settings.POOTLE_SCORE_COEFFICIENTS[
+    'SUGGESTION_REJECT']
+SCORE_CHECK_UNMUTED = settings.POOTLE_SCORE_COEFFICIENTS['CHECK_MUTED']
+SCORE_CHECK_MUTED = settings.POOTLE_SCORE_COEFFICIENTS['CHECK_UNMUTED']
+SCORE_COMMENT_UPDATED = settings.POOTLE_SCORE_COEFFICIENTS['COMMENT_UPDATED']
+SCORE_TARGET_UPDATED = settings.POOTLE_SCORE_COEFFICIENTS['TARGET_UPDATED']
+SCORE_STATE_TRANSLATED = settings.POOTLE_SCORE_COEFFICIENTS['STATE_FUZZY']
+SCORE_STATE_FUZZY = settings.POOTLE_SCORE_COEFFICIENTS['SUGGESTION_ADD']
+SCORE_STATE_UNFUZZY = settings.POOTLE_SCORE_COEFFICIENTS['STATE_UNFUZZY']
+SCORE_STATE_FUZZY_TRANSLATED = settings.POOTLE_SCORE_COEFFICIENTS[
+    'STATE_FUZZY_TRANSLATED']
+SCORE_STATE_UNTRANSLATED = settings.POOTLE_SCORE_COEFFICIENTS[
+    'STATE_UNTRANSLATED']
+SCORE_SOURCE_UPDATED = settings.POOTLE_SCORE_COEFFICIENTS['SOURCE_UPDATED']
```

Comment:
we need to be sure that all params are in place. if we set constants here it won't allow to start server without all coeffs.